app/inputs/transaction_chosen_old_input.rb

```diff
@@ -0,0 +1,63 @@
+# This class is deprecated in favor of TransactionChosenInput. This class
+# is still being used by BulkEmailController for searching.
+class TransactionChosenOldInput < SimpleForm::Inputs::Base # CollectionSelectInput
```

Comment:
I think we can switch this out when we do some other cleanup to the new views (like getting rid of extraneous instance variables).